src/scss/components/_carousel.scss

```diff
@@ -13,15 +16,17 @@
     @include col(4, 6);
   }
 
+  display: flex;
+  align-items: flex-end;
+
   &-overlap {
     z-index: var(--z-index-mid);
+    position: relative;
+    top: -100%;
```

Comment:
Don't position the overlap using absolute values. Use `margin-top` with negative value. Like this:
```
margin-top: -28.8rem; // Desktop
margin-top: -24.3rem; // Mobile
```

Reply:
Ready, thanks.